gl2csv.py

```diff
@@ -70,42 +82,17 @@ def main():
             else:
                 bodys = [x.body.lower() for x in notes if isinstance(x.body, (str, bytes)) and 'moved' in x.body.lower()]
                 if bodys:
-                    continue   # ignore all issues not ment to be here
+                    continue   # ignore all issues not meant to be here
                 else:
                     print('No closing date found {0!s}'.format(issue))
                     print 
                     print(bodys)
                     d['closed_at'] = d['updated_at']
                     closed_date = iso8601.parse_date(d['updated_at'].decode('utf-8'))
-            if True:  # closed_date.replace(tzinfo=None) > datetime.datetime(2017,4,1):
-                d['closed_week'] = '{:4d}-{:2d}'.format(closed_date.year, closed_date.month)
-                # d['closed_week'] = '{:4d}-{:2d}'.format(*closed_date.isocalendar()[:2])
-            else:
-                d['closed_week'] = ''
         else:
             d['closed_at'] = ''
-            d['closed_week'] = ''
-        #writer.writerow(d)
-        all_issues.append(d)
-        if d['closed_week']:
-            if d['closed_week'] not in weeks:
-                weeks[d['closed_week']] = 0
-            weeks[d['closed_week']] += 1
```

Comment:
Please don't remove. Grouping closed issues by week is important for my reporting.